### ISSUE TITLE
Ability to interpolate HTML inside text blocks

### ISSUE THREAD

**Issue by darthdeus** [slime-lang/slime#119]
Currently

```
p
  | Hello #{link "world", to: "/foo"}
```

will result in an error

```
protocol String.Chars not implemented for {:safe, [60, "a", [[32, "href", 61, 34, "/world", 34]], 62, "Hello", 60, 47, "a", 62]}
```

**Comment by doomspork** [comment]
Thanks @darthdeus!  Want to take a shot at implementing this?

**Comment by Rakoth** [comment]
@darthdeus could you confirm it is fixed with current master?

**Comment by Rakoth** [comment]
fixed by #130
